app/scripts/templates/settings/recovery_codes.mustache

```diff
@@ -0,0 +1,43 @@
+<div id="recovery-codes">
+    <section class="modal-panel">
+        <p class="modal-success">{{#t}}Two-step authentication enabled{{/t}}</p>
+        <div class="error"></div>
+
+        <h2>{{#t}}Recovery codes{{/t}}</h2>
+
+        <p class="description">
+            <span class="note">{{#t}}NOTE:{{/t}}</span>
+            {{#t}}
+                These codes will only be shown once. Store them in a safe place in the event you lose access to your mobile device.
+            {{/t}}
+        </p>
+
+        <form novalidate>
+            <div id="recovery-code-container">
+                {{#recoveryCodes}}
+                    <div class="recovery-code">{{code}}</div>
+                {{/recoveryCodes}}
+            </div>
+            <div class="link-options">
+                <a class="replace-codes-link">{{#t}}Replace codes{{/t}}</a>
```

Comment:
Would it be more clear to say "Generate new recovery codes"?

Reply:
That seems reasonable to me, @ryanfeeley objections?